Packs/TAXIIServer/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### TAXII Server
+- Added localhost as a host for WSGIServer creation
```

Comment:
You can use `demisto-sdk doc-review --templates` to write the RN according to our notation or go to https://xsoar.pan.dev/docs/documentation/release-notes